Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -317,7 +323,35 @@ def get_passed_mins(start_time, end_time_str):
 ''' COMMAND SPECIFIC FUNCTIONS '''
 
 
-def init_rtr_session(host_ids: list) -> str:
+def init_rtr_single_session(host_id: str) -> str:
+    """
+        Start a session with single host.
+        :param host_id: Host agent ID to initialize a RTR session on.
+        :return: The session ID to execute the command on
+    """
+    endpoint_url = '/real-time-response/entities/sessions/v1'
+    body = json.dumps({
+        'device_id': host_id
+    })
+    response = http_request('POST', endpoint_url, data=body)
+    resources = response.get('resources')
+    return resources[0].get('session_id') if isinstance(resources,list) and len(resources) != 0 else None
```

Comment:
need also to check `resources[0]` is of type `dict`
lets not make it a 1-liner for it to be better readable